src/SFA.DAS.EAS.Web/Controllers/EmployerAccountTransactionsController.cs

```diff
@@ -15,25 +15,40 @@ public class EmployerAccountTransactionsController : BaseController
     {
         private readonly EmployerAccountTransactionsOrchestrator _accountTransactionsOrchestrator;
 
-        public EmployerAccountTransactionsController(IOwinWrapper owinWrapper, IFeatureToggle featureToggle, 
+        public EmployerAccountTransactionsController(IOwinWrapper owinWrapper, IFeatureToggle featureToggle,
             EmployerAccountTransactionsOrchestrator accountTransactionsOrchestrator, IMultiVariantTestingService multiVariantTestingService,
-            ICookieStorageService<FlashMessageViewModel> flashMessage) 
-            : base(owinWrapper, featureToggle,multiVariantTestingService,flashMessage)
+            ICookieStorageService<FlashMessageViewModel> flashMessage)
+            : base(owinWrapper, featureToggle, multiVariantTestingService, flashMessage)
         {
             _accountTransactionsOrchestrator = accountTransactionsOrchestrator;
         }
 
-        //TODO: Review this once we know how the URL should look
-        [Route("balance")]
+        [Route("finance")]
         public async Task<ActionResult> Index(string hashedAccountId)
         {
-            return await Index(hashedAccountId, 0, 0);
+            var transactionViewResult = await _accountTransactionsOrchestrator.GetFinanceDashboardViewModel(hashedAccountId, 0, 0, OwinWrapper.GetClaimValue(ControllerConstants.SubClaimKeyName));
+
+            if (transactionViewResult.Data.Account == null)
```

Comment:
Will this code actually get called if there's a problem with the account hash or user validation?

Reply:
Account is retrieved in the mediator and if not found empty view model returned